app/domain/SlpFileWriter.js

```diff
@@ -2,18 +2,27 @@ import _ from 'lodash';
 import fs from 'fs-extra';
 import path from 'path';
 import moment from 'moment';
+import OBSWebSocket from 'obs-websocket-js'
 
 export default class SlpFileWriter {
   static commands = {
     CMD_RECEIVE_COMMANDS: 0x35,
+    CMD_GAME_START: 0x36,
     CMD_RECEIVE_POST_FRAME_UPDATE: 0x38,
     CMD_RECEIVE_GAME_END: 0x39,
   }
 
-  constructor(folderPath, onFileStateChange) {
+  constructor(folderPath, onFileStateChange, obsIP, obsSourceName) {
```

Comment:
I think id prefer is the first arg was changed to an object called settings which would have fields for folderPath, obsIP, etc. This is starting to be a lot of args